pkg/apiserver/apic.go

```diff
@@ -431,6 +431,8 @@ func (a *apic) PullTop() error {
 		return nil
 	}
 
+	log.Printf("Starting community-blocklist update")
```

Comment:
If it's the same as Infof, please use it. Otherwise there can be confusion between stdout/stderr